doc/coq.rst

```diff
@@ -158,7 +158,9 @@ Limitations
 .. _limitation-mlpack:
 
 - A ``foo.mlpack`` file must the present in directories of locally defined
-  plugins for things to work. This is a limitation of ``coqdep``.
+  plugins for things to work. This is a limitation of ``coqdep``. See the
```

Comment:
You can be more precise if you want "coqdep will recognize a plugin by looking at the existence of an .mlpack file, as it can't access (for now) Dune's library database"